topology/University of Illinois at Urbana Champaign/ICC-SLATE-HTC/UIUC-SLATE-HTC.yaml

```diff
@@ -0,0 +1,25 @@
+Production: true
+SupportCenter: Community Support Center
+GroupDescription: This is the UIUC High-Throughput Computing cluster used to support HTC users at UIUC and OSG opportunistically.
+GroupID: 
+Resources:
+  SLATE_US_UIUC_HTC:
+    Active: true
+    # Description is a long description of the resource; may be multiple lines
```

Comment:
```suggestion
```

Not really a big deal, but i think we can leave out this comment describing the purpose of the Description field.